src/core/BufferAttribute.js

```diff
@@ -310,6 +311,14 @@ BufferAttribute.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	discard: function () {
+
+		var oldArray = this.array;
+
+		this.array = new oldArray.constructor( 1 ); // create dummy minimal length TypedArray
```

Comment:
A rename fine, but the only discards I can see are in shaders, are you thinking of dispose()? 
I'll rename anyway.